ctapipe/tools/bokeh/file_viewer/main.py

```diff
@@ -392,5 +392,13 @@ def on_dl1_widget_change(self, _, __, ___):
                 self._updating_dl1 = False
 
 
-exe = BokehFileViewer()
-exe.run()
+def main():
+    exe = BokehFileViewer()
+    exe.run()
+
+
+if 'bk_script' in __name__:
```

Comment:
To run a bokeh server you have to execute `bokeh serve` on the script instead of `python`. When you do this, `__name__` equals something like "bk_script_dc88af7ef51b4e0d85854896dd662ed0", where the third term seems to be random.

Reply:
However, there may be a way to avoid having to use `bokeh serve` if I can understand how to utilise this: http://bokeh.pydata.org/en/latest/docs/user_guide/server.html#embedding-bokeh-server-as-a-library